Validation/Configuration/python/mtdSimValid_cff.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+# MTD validation sequences
```

Comment:
This file should be in `Validation/MtdValidation/python`.

Reply:
I've put this file here for consistency with all the other systems. Tracker, ECAL, HCAL, Muons. HGCAL, GEM have their corresponding config file here.